aips/objects.md

```diff
@@ -90,60 +91,50 @@ Each object is stored in its own address or object id. To ensure unique Ids, the
 
 ### Capabilities
 
+- `ObjectId`
+    - A pointer to an object, a type-safe wrapper around address
+    - Abilities: `copy, drop, store`
+    - Data layout `{ inner: address }`
 - `CreatorRef`
     - Provided to the object creator at creation time
     - Can create any other ref type
     - Abilities: `drop`
-    - Data layout `{ self: address }`
-- `OwnerRef`
-    - Represents ownership
-    - There can be many for a single object
-    - They can be aggregated
-    - Abilities: `store`
-    - Data layout: `{ self: address, amount: u64 }`
-- `MintRef`
-    - Has the ability to generate `OwnerRef`
-    - Abilities: `store`
-    - Data layout: `{ self: address }`
+    - Data layout `{ self: ObjectId }`
 - `DeleteRef`
     - Can be used to remove an object from the `ObjectGroup`
     - There can be many for a single object
     - They cannot be aggregated
```

Comment:
Why many/not aggregatable? Are any other aggregatable? Do we not expect other ref types to have multiple for an object?

Reply:
I should just delete this...